grafana-plugin/src/models/filters/filters.helpers.ts

```diff
@@ -7,3 +11,13 @@ export const getApiPathByPage = (page: string) => {
     }[page] || page
   );
 };
+
+export const normalizeFilters = (filters: FiltersValues, filterOptions: FilterOption[]) => {
+  const normalizeFilters = { ...filters };
```

Comment:
This const doesn't represent a function so it shouldn't have a verb in the name. Also we don't do any normalization here, we just create a shallow copy. You could name this const `filtersCopy`